.github/CODEOWNERS

```diff
@@ -5,6 +5,8 @@
 # Default fallback
 *    @abcxyz/breakglass
 
+*    @abcxyz/infrastructure-team
```

Comment:
Why do we need this? (curious)

Reply:
No one other than breakglass owned the workflow file so there was no one who could be an approving reviewer without adding them to breakglass.

Reply:
Ah!